## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 420704`

Logs: https://github.com/Aleksanaa/nixpkgs-review-gha/actions/runs/15942327880


---
### `x86_64-linux` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ladybird</li>
  </ul>
</details>


---
### `aarch64-linux` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ladybird</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>ladybird</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>ladybird</summary>
<pre>    
    Run Build Command(s): /nix/store/npg4pj5np9rr812mr36j7bam8drxq6py-ninja-1.12.1/bin/ninja -v cmTC_2f673
    [1/2] /nix/store/cpjrfk28hkjl7qxxamsc0v9sndj41wc1-clang-wrapper-19.1.7/bin/clang   -isysroot /nix/store/mp4983pbkvrc0vd60ihlbd0ydbjcbpb3-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -MD -MT CMakeFiles/cmTC_2f673.dir/testCCompiler.c.o -MF CMakeFiles/cmTC_2f673.dir/testCCompiler.c.o.d -o CMakeFiles/cmTC_2f673.dir/testCCompiler.c.o -c /tmp/nix-build-ladybird-0-unstable-2025-06-27.drv-0/source/build/CMakeFiles/CMakeScratch/TryCompile-jxgp9I/testCCompiler.c
    [2/2] : && /nix/store/cpjrfk28hkjl7qxxamsc0v9sndj41wc1-clang-wrapper-19.1.7/bin/clang -isysroot /nix/store/mp4983pbkvrc0vd60ihlbd0ydbjcbpb3-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/cmTC_2f673.dir/testCCompiler.c.o -o cmTC_2f673   && :
    FAILED: cmTC_2f673 
    : && /nix/store/cpjrfk28hkjl7qxxamsc0v9sndj41wc1-clang-wrapper-19.1.7/bin/clang -isysroot /nix/store/mp4983pbkvrc0vd60ihlbd0ydbjcbpb3-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/cmTC_2f673.dir/testCCompiler.c.o -o cmTC_2f673   && :
    ld: library not found for -lGL
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ninja: build stopped: subcommand failed.
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:30 (project)


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>ladybird</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>ladybird</summary>
<pre>    
    Run Build Command(s): /nix/store/ssj3qa52h5ajvzckyhnpv1rz7zc68a1x-ninja-1.12.1/bin/ninja -v cmTC_95d30
    [1/2] /nix/store/1g645wydyvns7dy8gy27dq0n3rbkd712-clang-wrapper-19.1.7/bin/clang   -isysroot /nix/store/p9kkihbhakgw209d02xq24vw5z8ff8bc-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -MD -MT CMakeFiles/cmTC_95d30.dir/testCCompiler.c.o -MF CMakeFiles/cmTC_95d30.dir/testCCompiler.c.o.d -o CMakeFiles/cmTC_95d30.dir/testCCompiler.c.o -c /tmp/nix-build-ladybird-0-unstable-2025-06-27.drv-0/source/build/CMakeFiles/CMakeScratch/TryCompile-stwUjZ/testCCompiler.c
    [2/2] : && /nix/store/1g645wydyvns7dy8gy27dq0n3rbkd712-clang-wrapper-19.1.7/bin/clang -isysroot /nix/store/p9kkihbhakgw209d02xq24vw5z8ff8bc-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/cmTC_95d30.dir/testCCompiler.c.o -o cmTC_95d30   && :
    FAILED: cmTC_95d30 
    : && /nix/store/1g645wydyvns7dy8gy27dq0n3rbkd712-clang-wrapper-19.1.7/bin/clang -isysroot /nix/store/p9kkihbhakgw209d02xq24vw5z8ff8bc-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -mmacosx-version-min=14.0 -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/cmTC_95d30.dir/testCCompiler.c.o -o cmTC_95d30   && :
    ld: library not found for -lGL
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ninja: build stopped: subcommand failed.
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:30 (project)


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

--- a/pkgs/by-name/la/ladybird/package.nix
+++ b/pkgs/by-name/la/ladybird/package.nix
@@ -35,17 +35,17 @@
 
 let
   # Note: The cacert version is synthetic and must match the version in the package's CMake
-  cacert_version = "2023-12-12";
+  cacert_version = "2025-05-20";
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "ladybird";
-  version = "0-unstable-2025-06-18";
+  version = "0-unstable-2025-06-27";
 
   src = fetchFromGitHub {
     owner = "LadybirdWebBrowser";
     repo = "ladybird";
-    rev = "86c8dbbf902d5c84bdc90cd35d19cc167d5848bd";
-    hash = "sha256-CUVKrbpwPDMHPMHQfNSOQLQmRv7Fy4H/xsglZJaPZlI=";
+    rev = "831ba5d6550fd9dfaf90153876ff42396f7165ac";
+    hash = "sha256-7feXPFKExjuOGbitlAkSEEzYNEZb6hGSDUZW1EJGIW8=";
   };
 
   postPatch = ''
@@ -201,5 +201,6 @@ stdenv.mkDerivation (finalAttrs: {
       "aarch64-darwin"
     ];
     mainProgram = "Ladybird";
+    broken = stdenv.hostPlatform.isDarwin;
   };
 })

